Felienne/hedy
# [FIX] correctly displays copy button on left of editor when RTL language

**Description**
In this PR we fix a small bug where the copy button on code snippets was always placed in the top-right corner. However, with RTL languages this overlays the text. We fix this by styling the button using the Tailwind `ltr:` and `rtl:` classes.
 
**Fixes**
This PR fixes #2082

**How to test**
Navigate to the code page and there should be no noticeable difference for LTR languages. Switch to Arabic and verify that the copy button is now correctly displayed in the top-left corner of the code snippets.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
